exchanges/orderbook/depth_test.go

```diff
@@ -46,7 +46,7 @@ func TestRetrieve(t *testing.T) {
 		pair:             currency.NewPair(currency.THETA, currency.USD),
 		asset:            "Silly asset",
 		lastUpdated:      time.Now(),
-		lastUpdateID:     007,
+		lastUpdateID:     007, // nolint:gocritic //  octal literal false positive
```

Comment:
Not a huge deal, but we can save on the nolint by using a better number, like 1337